samples/flight_control/find_child_cid.py

```diff
@@ -0,0 +1,84 @@
+"""Identify CID for a child host.
+
+ _______ __ __       __    __    _______             __              __
+|   _   |  |__.-----|  |--|  |_ |   _   .-----.-----|  |_.----.-----|  |
+|.  1___|  |  |  _  |     |   _||.  1___|  _  |     |   _|   _|  _  |  |
+|.  __) |__|__|___  |__|__|____||.  |___|_____|__|__|____|__| |_____|__|
+|:  |         |_____|           |:  1   |
+|::.|                           |::.. . |
+`---'                           `-------'
+
+
+This solution leverages the Flight Control and Hosts APIs.
+Recommended version: FalconPy v0.8.6+
+
+Creation date: 01.07.2021 - jshcodes@CrowdStrike
+
+You will need the following scopes on your API keys:
+    Flight Control: READ
+    Hosts: READ
+"""
+import argparse
+from falconpy import Hosts, FlightControl, OAuth2
+
+
+def parse_command_line() -> object:
+    """Parse the command line for inbound configuration parameters."""
+    parser = argparse.ArgumentParser(description="Falcon X Artifact retrieval")
```

Comment:
@jshcodes may be leftover

Reply:
Doh! Copy paste got me... resolving now.